backend/lib/index.js

```diff
@@ -1,22 +1,26 @@
 const cors = require("cors");
-const passport = require("./plugins/passport");
-const version = require("./endpoints/version");
+const fastify = require("fastify");
+
 const auth = require("./endpoints/auth");
 const geo = require("./endpoints/geo");
 const posts = require("./endpoints/posts");
-const user = require("./endpoints/user");
+const users = require("./endpoints/users");
+const version = require("./endpoints/version");
 
-function createServer(app) {
+module.exports = function createApp(config) {
+  const app = fastify({
+    logger: true,
+  });
+
+  app.register(require("./plugins/mongoose-connector"), config.mongo);
```

Comment:
The same, still don't like this change

Reply:
"Not liking something" is not  very useful code review feedback.
You can state that you have a different personal preference, but it's better to back up your opinion with some explanation for your reasoning.
There are many useful resources to improve your code review comments, like this one from Google https://google.github.io/eng-practices/review/reviewer/comments.html